app/controllers/games_controller.rb

```diff
@@ -1,21 +1,26 @@
 class GamesController < ApplicationController
-  before_action :set_game, only: %i[show edit update destroy]
+  before_action :set_game, only: %i[edit update destroy]
 
   def index
     @sort = params[:sort] || Game.default_sort_column
     @sort_dir = params[:sort_dir] || Game.default_direction
 
-    @games = Game.includes(:game_players).all.sort_table(@sort, @sort_dir)
+    @games = Game.includes(:rounds, :game_players).all.sort_table(@sort, @sort_dir)
```

Comment:
I think this would work better with `includes(game_players: :rounds)`. Wdyt?

Reply:
If we do that we will have a n+1 problem on the rounds column.